Extensions/Multiplayer/messageManager.ts

```diff
@@ -2123,10 +2175,13 @@ namespace gdjs {
       createVariableOwnerChangedMessageNameFromChangeVariableOwnerMessage,
       handleChangeVariableOwnerMessagesReceived,
       // Custom messages.
-      sendMessage,
-      hasMessageBeenReceived,
-      getMessageData,
+      sendCustomMessage,
+      sendCustomVariableMessage,
+      hasCustomMessageBeenReceived,
+      getCustomMessageData,
+      getCustomVariableMessageData,
```

Comment:
getCustomMessageVariableData

(it's the Message that is custom, not the variable)